decidim-meetings/lib/decidim/meetings/engine.rb

```diff
@@ -133,6 +133,14 @@ class Engine < ::Rails::Engine
           end
         end
       end
+
+      initializer "decidim_meetings.authorization_transfer" do
+        Decidim::AuthorizationTransfer.register(:meetings) do |transfer|
+          transfer.move_records(Decidim::Meetings::Meeting, :decidim_author_id)
```

Comment:
I also think for consistency it would be good to keep here, if we are anyways transferring data, so it may feel weird that some data was not actually transferred.

In my opinion this feature is more about data recovery in general, as we know that data was originally created by the same person based on the authorization.